examples/todo_app_basic/src/main.rs

```diff
@@ -0,0 +1,86 @@
+use leptos::*;
+
+fn main() {
+    mount_to_body(|cx| view! { cx, <App /> });
+}
+
+#[derive(Eq, Hash, PartialEq, Clone, Debug)]
+struct TodoItem {
+    id: u32,
+    name: String,
+}
+
+#[derive(Copy, Clone)]
+struct TodoGenerator {
```

Comment:
This should be a `struct TodoId` that has a `fn next(&self)` or so but you don't want to have a `TodoGenerator`.